node/sc/subbridge.go

```diff
@@ -300,6 +302,13 @@ func (sb *SubBridge) APIs() []rpc.API {
 			Service:   sb.APIBackend,
 			Public:    true,
 		},
+		{
+			Namespace:  "subbridge",
+			Version:    "1.0",
+			Service:    sb.PrivilegedAPIBackend,
+			Privileged: true,
+			Public:     true,
```

Comment:
What is the difference between `Public` and `Privileged`?
It seems there is a feature where an API is only exposed via IPC if `api.Public == false`.
https://github.com/klaytn/klaytn/blob/dev/networks/rpc/endpoints.go

Reply:
There's another condition near that `api.Public`. See (https://github.com/klaytn/klaytn/blob/dev/networks/rpc/endpoints.go#L37). The `whitelist` value must be empty not to register the APIs. The `whitelist` becomes empty if no API is given in the option of `RPC_LIST` in `*.conf`. Thus, this intention is to enable the namespace of APIs that sets `Public` to true if there's no any given pre-set RPCs.

Reply:
Understood. Node operator can choose to expose non-`Public` RPCs with `RPC_LIST`. But we want to absolutely prohibit exposing `Privileged` RPCs whatsoever.

Reply:
Yes. The privileged APIs in the specific namespace of RPC are not exposed to HTTP scheme, FastHTTP scheme, gRPC scheme, WS scheme, and FastWS scheme, but are allowed to only IPC scheme. 

* RPC_API (Not `RPC_LIST`. I'm sorry I confused the option name)